tflite2onnx/op/reshape.py

```diff
@@ -72,13 +73,45 @@ def parse(self):
 
         self.setParsed()
 
+    def fakeTranspose(self):
+        # If the input (or output) tensor of Reshape
+        # holds a special data layout semantic,
+        # we need to insert Transpose before (or after) the Reshape operator
+        # https://github.com/jackwish/tflite2onnx/issues/28
+
+        # assert(self.status.initialized)
+        to_transpose = self.inputs[0]
+
+        transposed_name = 'TFLITE2ONNX_Transposed_%s' % to_transpose.name
+        transposed = self.TFactory.getWithRef(to_transpose, transposed_name, True)
+        transposed.shape = [to_transpose.shape[p] for p in [0, 2, 3, 1]]
+        transposed.setParsed()
+
+        # Construct the additional transpose
+        trans = Transpose(self.TFactory, -1)
+        trans.attrs['perm'] = np.array([0, 2, 3, 1])
+
+        trans.inputs.append(to_transpose)
+        to_transpose.replaceConsumer(self, trans)
+        self.replaceInput(to_transpose, transposed)
+
+        trans.outputs.append(transposed)
+        transposed.addProducer(trans)
+        transposed.addConsumer(self)
+        trans.setParsed()
+
+        self.pre.append(trans)
+
     def propagatableTensors(self):
         return list()
 
     def transform(self):
+        i = self.inputs[0]
+        if i.layout is not None:
```

Comment:
Could be `if i.layout`. I know I have some place that written like `if i.layout is not None`...